### PR TITLE
Add support for pages create or improved on Wikidata

### PR DESCRIPTION
**Pull request by samwilson**
This adds stats for Wikidata, counting the same stats as for
Wikipedia but renaming them.

Bug: https://phabricator.wikimedia.org/T190456

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -91,8 +91,10 @@
 	"pager-older-n": "{{PLURAL:$1|Older $1|Older $1}}",
 	"pages-created": "Pages created",
 	"pages-created-desc": "A content page is a page in either the main or file namespace.",
+	"pages-created-wikidata": "Items created",
 	"pages-improved": "Pages improved",
 	"pages-improved-desc": "A content page is a page in either the main or file namespace.",
+	"pages-improved-wikidata": "Items improved",
```
Comment (MusikAnimal) [inline i18n/en.json]: don't forget qqq :)

Comment (samwilson) [reply i18n/en.json]: Good point! Should we think about adding a check for that to the CI? A codesniff should be added for it to Intuition maybe?

Comment (MusikAnimal) [reply i18n/en.json]: That would be magnificent! If you know how to do it, by all means :) I wasn't sure if you could easily program in your own sniff rules like that.

Comment (MusikAnimal) [reply i18n/en.json]: That would be magnificent! If you know how to do it, by all means :) I wasn't sure if you could easily program in your own sniff rules like that.

**Comment by samwilson** [comment]
This might be ready for review, if you've got a mo.

**Comment by MusikAnimal** [comment]
Overall looks good! There's no test coverage for `EventProcessor::setItemsCreatedOrImprovedOnWikidata()`, but I guess that's okay because it's exactly the same as `setItemsCreatedOrImprovedOnWikidata()`. On that note, should we re-use the same method? Or add `@codeCoverageIgnore` (with an explanatory comment) to the Wikidata method so our test coverage isn't impacted.

Another question: I don't know Wikidata very well... are we sure only items (starting with `Q`) are in the mainspace?

**Comment by samwilson** [comment]
Good point about tests! I totally did start there, but got a tiny bit lazy. Done now.

And yep, all mainspaces pages on WD are items.

**Pull request merged**